apps/root/src/common/components/networth-number/index.tsx

```diff
@@ -48,7 +50,7 @@ const NetWorthNumber = ({
   const [totalInteger, totalDecimal] = fixedWorth.split('.');
 
   return (
-    <StyledNetWorth variant={variant}>
+    <StyledNetWorth variant={variant} $light={light}>
```

Comment:
Up to now, I've found two cases where net worth number color changes (only integer part)

Reply:
they must have been overlooked by our UX designer, could you check with him if that is correct?

Reply:
Checking!

Reply:
Colors for this component are now updated in the figma, and there are two use cases: Using this for fiat or crypto numbers (Fiat have $ and light-color decimals)